CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ Here is a template for new release sections
 
 ### Added
 ### Changed
+- renewable waste fuel (and subclasses) -> biogenic waste fuel (and subclasses); biofuel (#1048)
```

Comment:
Im not a fan of very detailed changelog notes. They should stay very simple. One can find more detailes either in the PR or the term trackers...